tools/mkmapdata/mkmapdata.sh

```diff
@@ -509,17 +484,28 @@ until ((nfile>${#INGRID[*]})); do
       echo "Skipping creation of ${OUTFILE[nfile]} as fast mode is on so skipping large files in NetCDF4 format"
    else
 
-      cmd="$mpirun $ESMF_REGRID --ignore_unmapped -s ${INGRID[nfile]} "
-      cmd="$cmd -d $GRIDFILE -m conserve -w ${OUTFILE[nfile]}"
-      if [ $type = "regional" ]; then
-        cmd="$cmd --dst_regional"
+      PATH_PREFIX=/glade/work/slevis
+      WD=${PATH_PREFIX}/git/mkmapdata_ocgis/tools/mkmapdata
+      SS_PATH=${WD}/subsets/spatial_subset.nc
+      rm -rf ${SS_PATH}
+      CHUNKDIR=${WD}/chunking
+      rm -rf ${CHUNKDIR}
+      OCLI_FILE=${PATH_PREFIX}/git_ocgis/ocgis/src/ocgis/ocli.py
+      CRWG="python ${OCLI_FILE} chunked-rwg"
```

Comment:
Some of these names are a bit short and cryptic. I get WD for Working Directory. And I suppose from context SS_PATH is Spatial Subset path. But, I'm not sure what CRWG means? What does the "chunked -rwg" option mean?

Reply:
I added this variable for the development script. It allows switching the CLI executable easily. For production code, I expect you could just use `ocli chunked-rwg` (chunked-rwg is the CLI subcommand).

Reply:
I'm mainly complaining about the names being too short to be meaningful and obvious (at least to me). So if it could be a little longer and more descriptive that would be useful. Or adding a comment that describes what it's doing and what its purpose is. But, typically it's better to just have a longer more descriptive name.

Reply:
:+1: Names are indeed a bit cryptic.

Reply:
Renamed CRWG to CLI_EXECUTABLE.
Renamed SS_PATH to SUBSETS_PATH.
Removed PATH_PREFIX and WD.